gitspy/Parseable.swift

```diff
@@ -9,7 +9,31 @@
 import Foundation
 
 protocol Parseable {
-    typealias A
-    func parse(data: AnyObject) -> A?
+    
+    init?(dictionary: [String: AnyObject])
+    init?(data: AnyObject)
```

Comment:
Thanks for the failable initializers, they probably make a lot of sense. But do we still need the init?(data: AnyObject)?